src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<Configuration status ="info">
+    <Appenders>
+        <Console name = "CONSOLE_OUT" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} %-5level %logger{36} - %msg%n"/>
+        </Console>
+        <File name="File" fileName="logs/app.log">
+            <PatternLayout>
+                <Pattern>%d %p %c %m%n </Pattern>
+            </PatternLayout>
+        </File>
+    </Appenders>
+    <Loggers>
+        <Logger name="mate.academy" level="debug" additivity="false">
+            <AppenderRef ref="File"/>
+            <AppenderRef ref="CONSOLE_OUT"/>
+        </Logger>
+        <Root level="debug">
```

Comment:
usually the root level is set to `info`

however in scope of this PR you may set it to `debug` in order to see your logs :)